packages/sdk/client/src/services/local-client-services.ts

```diff
@@ -73,12 +87,28 @@ export class LocalClientServices implements ClientServicesProvider {
   private readonly _ctx = new Context();
   private readonly _params: ClientServicesHostParams;
   private _host?: ClientServicesHost;
+  signalMetadataTags: any = {
+    runtime: 'local-client-services',
+  };
 
   @trace.info()
   private _isOpen = false;
 
   constructor(params: ClientServicesHostParams) {
     this._params = params;
+    // TODO(nf): extract
+    if (typeof window === 'undefined') {
+      // TODO(nf): collect ClientServices metadata as param?
+      this.signalMetadataTags.origin = 'undefined';
+    } else {
+      // SocketSupply native app
+      if ((globalThis as any).__args) {
+        this.signalMetadataTags.origin = 'native';
```

Comment:
fwiw I think window.location.origin will still work here and will give an indication of _which_ app it is

Reply:
ok, i still can't get the app to run so I'll just have to take your word for it :)